src/wpilib/robot-states.md

```diff
@@ -0,0 +1,36 @@
+# Robot States
+
+Robots programmed with WPILib can be in one of four states:
+
+- **Disabled**:
+  The robot does not accept user input and shouldn't move mechanisms.
+  It is safe for humans to be around the robot while it is disabled.
+  This is the default state.
+- **Teleoperated**:
+  The robot accepts user input and moves mechanisms.
+  This is the most common state used while testing,
+  and the teleoperated period of the FRC game is spent in this state.
+- **Autonomous**:
+  The robot does not accept user input, and instead goes through a sequence of preprogrammed actions.
+  This state is active for the first 15 seconds in an FRC match.
+- **Emergency Stopped (E-Stopped)**:
+  This is a special state which is identical to the disabled state,
+  except that the bot cannot be re-enabled after it has been emergency stopped.
```

Comment:
Maybe this could include the fact that the space bar (I think) triggers it? Something like "The space bar can be used to emergency stop the robot, forcing it into the disabled state until the robot is restarted.